docs/guide/save_format.rst

```diff
@@ -32,14 +32,6 @@ This format allows skipping elements in the file, i.e. we can skip deserializing
 broken/non-serializable.
 This can be done via ``custom_objects`` argument to load functions.
 
-.. note::
```

Comment:
why was that removed?
you can encounter issues for other reasons, for instance different cloudpickle version...

Reply:
btw, I know cloudpickle is not meant for long-term storage, that's why we only use it as a backup when pickle fails.

Reply:
Indeed